header.php

```diff
@@ -56,6 +56,70 @@
   <section id="main-container">
 
   <!-- start content -->
-  <header id="header">
-    <h1><a href="<?php echo home_url(); ?>"><?php bloginfo('name'); ?></a></h1>
+  <header id="header" class="u-fixed">
+    <h1 class="u-visuallyhidden"><?php bloginfo('name'); ?></h1>
+    <nav class="menu">
+      <ul class="main-menu row">
+<?php 
+$posts = get_posts();
+
+if (count($posts) > 0) {
+?>
+        <li>
+          Studio
+          <ul class="sub-menu">
+<?php 
+  foreach($posts as $post) {
+    setup_postdata($post);
+?>
+            <li class="studio-list-item">
+              <a href="<?php the_permalink(); ?>"><?php the_title(); ?></a>
+            </li>
+<?php 
+  }
+  wp_reset_postdata();
+?>
+          </ul>
+        </li>
+<?php 
+}
+
+$cat_array = get_categories();
+
+if ($cat_array) {
+?>
+        <li>
+          Sort
```

Comment:
looks to me from the design that the toggles Studio Sort and Info exist in a row in the header. We can even use flex grid here and just manually have a container set to 98% or whatever rather than the normal `.container` width